home/jobs/OMERO-test-integration/config.xml

```diff
@@ -93,6 +94,11 @@ for x in *.tar.gz; do
     pip install -U $x
 done
 
+# Cause CLI plugin tests to be run against the relevant version.
+for x in omero-cli-duplicate; do
```

Comment:
Making this easily extensible and looping over the installation of CLI plugins with associated integration tests in `ome/openmicroscopy` makes a lot of sense.

Has this change been tested in a daily CI execution or is it worth re-running merge-ci OMERO-test-integration to ensure the logic does what we want?

Reply:
It isn't been tested in a daily CI execution.

Reply:
Altered latest, merge configs by hand, they seem to be happy so far.